catalog/utilities/media_props_gen/generate_media_properties.py

```diff
@@ -0,0 +1,137 @@
+"""Automatic media properties generation."""
+
+import logging
+from dataclasses import dataclass
+from pathlib import Path
+
+from column_parser import parse_python_columns
+from db import MediaType, create_db_props_dict
+from md import Md
+
+
+log = logging.getLogger(__name__)
+# Silence noisy modules
+logging.getLogger("common.storage.media").setLevel(logging.WARNING)
+
+# Constants
+PARENT = Path(__file__).parent
+DOC_MD_PATH = PARENT / "media_properties.md"
+SOURCE_MD_PATH = PARENT / "media_props.md"
+
+PREAMBLE = open(Path(__file__).parent / "preamble.md").read()
```

Comment:
We have a `postamble` file defined as well, but no variable for it. Should it be referenced here or removed?

Reply:
I added the POSTAMBLE here and in the document.